### PR TITLE
Added recommendations file for Belfast.

### PR DESCRIPTION
**Pull request by 0mpurdy**

### CODE REVIEW

**Comment by 0mpurdy** [comment]
closes #9

File: 0mpurdy-belfast [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Recommendations for Belfast
+
+### Places to eat
```
Comment (jajain) [inline 0mpurdy-belfast]: Do you have a recommendation for places where I can find vegetarian dishes?

Comment (0mpurdy) [inline 0mpurdy-belfast]: I'm sure I could add some 😄

Comment (0mpurdy) [inline 0mpurdy-belfast]: Done now - want to approve changes?

Comment (jajain) [inline 0mpurdy-belfast]: Approved :)

**Pull request merged**